test/e2e/cluster_upgrade_runtimesdk.go

```diff
@@ -89,21 +89,27 @@ type clusterUpgradeWithRuntimeSDKSpecInput struct {
 	// Allows to inject a function to be run after the cluster is upgraded.
 	// If not specified, this is a no-op.
 	PostUpgrade func(managementClusterProxy framework.ClusterProxy, workloadClusterNamespace, workloadClusterName string)
+
+	// ExtensionNamespace is the namespace where the service for the Runtime SDK is located
+	// and is used to configure in the test-namespace scoped ExtensionConfig.
```

Comment:
considering ExtensionConfig is a cluster-wide object this comment is a bit confusing (same below)

Reply:
Good pooint, wdyt about `ExtensionConfigServiceNamespace`?

Reply:
Sounds like a good improvement. My comment was mostly about rephrasing the godoc comment a bit though :D (but an additional rename also sounds good)